examples/just-the-basics/src/components/Layout.astro

```diff
@@ -0,0 +1,52 @@
+---
+const { title } = Astro.props;
```

Comment:
Thoughts on adding `export interface Props {}` to this file? I bet @Princesseuh would love if users had typed props by default.

Reply:
Love this! Adding asap